recipes-bsp/xt-rpi-u-boot-scr/xt-rpi-u-boot-scr.bb

```diff
@@ -30,6 +30,8 @@ XENPOLICY_IMAGE ?= "xenpolicy"
 XENPOLICY_IMG_ADDR ?= "0x2300000"
 UBOOT_BOOT_SCRIPT ?= "boot.scr"
 UBOOT_BOOT_SCRIPT_SOURCE ?= "boot.cmd"
+DOM0_BOOTARGS ?= "console=hvc0 earlycon=xen earlyprintk=xen clk_ignore_unused root=\/dev\/ram0"
+DOMD_BOOTARGS ?= "console=ttyAMA0 earlycon=xen earlyprintk=xen clk_ignore_unused root=\/dev\/mmcblk0p2 rootfstype=ext4 rootwait"
```

Comment:
So, this can be passed in .yaml right?

Reply:
Yes, it is possible

Reply:
It works but every slash should be escaped with 5 backslash in yaml. Something like this:
```
        - [DOM0_BOOTARGS, "console=hvc0 earlycon=xen earlyprintk=xen clk_ignore_unused root=\\\\\/dev\\\\\/ram0"]
```

Reply:
Actually 4 backslashes will enough